src/Views/Signin/index.js

```diff
@@ -1,9 +1,9 @@
-import React from 'react';
-import SigninForm from '../../Components/Signin/SigninForm';
-import SigninHeader from '../../Components/Signin/SigninHeader';
-import SigninHero from '../../Components/Signin/SigninHero';
+import React from './node_modules/react';
+import SigninForm from '../../components/signin/signin-form';
+import SigninHeader from '../../components/signin/signin-header';
+import SigninHero from '../../components/signin/signin-hero';
```

Comment:
Change import name to be with uppercase `C` or change folder name to be with lowercase `c`